CDTDatastore/query/CDTQIndex.h

```diff
@@ -65,7 +79,10 @@ extern NSString *const kCDTQTextType;
  * @param indexSettings the indexSettings to compare to as an NSString
  * @return YES/NO - whether there is a match
  */
-- (BOOL)compareIndexTypeTo:(NSString *)indexType withIndexSettings:(NSString *)indexSettings;
+- (BOOL)compareIndexTypeTo:(NSString *)indexType
+         withIndexSettings:(NSString *)indexSettings __deprecated;
+
+- (BOOL)compareToIndexType:(CDTQIndexType)indexType withIndexSettings:(NSString *)indexSettings;
```

Comment:
This method should probably have its own doc comments (largely copied from the deprecated method).